tests/test_pytest/test_output_checkers.py

```diff
@@ -0,0 +1,54 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Output Checker plugins."""
```

Comment:
Also, @AJThorpe @evrose54 @jsolbrig , do you think we want to organize the test_pytest directory differently at all, as we are getting things set up to add more and more pytests?

test_pytest/test_all_plugins
* from any interface ?
test_pytest/test_plugin_schema
* This is specifically testing the schema, not the plugins themselves
test_pytest/test_utilities
* things like dtree, etc ?

Reply:
In my opinion, I'd split up the directory by interfaces. Ie:
  - test_pytest/
      - readers/
      - output_checkers/
      - ...

Reply:
That way, if we had individual tests for plugins that we didn't want to combine in one file, we could have an organized directory pointing to each correlated test.

Reply:
Yes, this sounds like a good structure to me. Directories for each interface.

Reply:
Do we want to have another level
test_pytest
* test_plugins
  * readers
  * output_checkers
* test_plugin_schema
  * ...
* test_utilities
  * xarray_utils
  * mpl_utils
  * ???

Reply:
I'll leave this up to Jeremy. Generally, I prefer a shorter directory structure, meaning I'd probably have schemas, modules, and yamls as top-level directories, then sort them appropriately by their plugin family/interface. Ie:
  - test_pytest/
      - modules/
          - algorithms
          - ...
      - yamls/
          - products
          - ...
      - schemas/
          - ...